.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
I would actually prefer if we made it a devDependency. It's preferable to have all of our dependencies in one place. Would that be OK?